python/util/view_access_level.py

```diff
@@ -2,4 +2,5 @@
 IN_DOMAIN_CONTEXT = 1
 LOGGED_IN_USER = 2
 MANAGED_DOMAIN = 3
-SUPER_USER_ONLY = 4
```

Comment:
better not to edit these kinds of constants -- just make new ones.

Reply:
agreed